sbt/src/sbt-test/project/unified/build.sbt

```diff
@@ -0,0 +1,12 @@
+lazy val root = (project in file("."))
+  .settings(
+    Global / cancelable := true,
+    ThisBuild / scalaVersion := "2.12.2",
```

Comment:
I can deprecate the use of `in` operator. I'm still not sure if we should remove in sbt 1.0 or wait till later.